src/components/login/facebook-login/facebook-login.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FacebookLogin from "react-facebook-login";
+import "../google-facebook-login.scss";
+import { FaFacebook } from "react-icons/fa";
+import ButtonNames from "../../../constants/ButtonNamesConstants";
+
+const { FACEBOOK_LOGIN } = ButtonNames;
+
+const LoginFacebook = ({ config }, props) => {
+  const responseFacebook = response => {
+    if (!response.email) {
+      props.setErrorAlert("Please add email to your facebook account!");
```

Comment:
Can all props be destructed inside function params

Reply:
Yes. Changed.